sbt-plugin/src/sbt-test/gen-java/01-gen-basic-java/build.sbt

```diff
@@ -1,4 +1,5 @@
 resolvers += Resolver.sonatypeRepo("staging")
+resolvers += Resolver.bintrayRepo("akka", "snapshots")
```

Comment:
All of these can probably be removed for now? Or do you want to keep them in there to simplify testing the next unreleased changes / improvements?

Reply:
I think it's kind of nice to keep them for when we want to test with snapshot versions in different PR's as well